master/internal/scheduler/cluster.go

```diff
@@ -341,6 +356,24 @@ func (c *Cluster) receiveAddTask(ctx *actor.Context, msg AddTask) {
 	}
 }
 
+func (c *Cluster) receiveStartTask(ctx *actor.Context, msg StartTask) {
+	task := c.tasksByHandler[ctx.Sender()]
+	if task == nil {
+		ctx.Log().WithField("address", ctx.Sender().Address).Errorf("unknown task trying to start")
```

Comment:
Sorry, my bad:
```suggestion
		ctx.Log().WithField("address", ctx.Sender().Address()).Errorf("unknown task trying to start")
```

Reply:
nice catch, thanks

Reply:
Heh, just canceling out the fact that I put a bad suggestion in the first place. :man_facepalming: